include/swift/AST/DiagnosticsSema.def

```diff
@@ -7497,12 +7497,12 @@ ERROR(wrap_invalid_attr_added_by_access_note, none,
 
 ERROR(cannot_default_generic_parameter_inferrable_from_another_parameter, none,
       "cannot use default expression for inference of %0 because it "
-      "is inferrable from parameters %1",
+      "is inferable from parameters %1",
```

Comment:
While "inferable" is a more common spelling, "inferrable" is used in some published works:

<img width="570" alt="Screenshot 2024-07-05 at 2 28 19 PM" src="https://github.com/swiftlang/swift/assets/931226/37b04747-21b3-4f3d-88e5-59b459eff4a4">

Swift is consistent enough in its use of "inferrable" that I'd be inclined to keep it.

Reply:
reverted all `inferrable` -> `inferable` changes